auth/saml.adoc

```diff
@@ -0,0 +1,224 @@
+
+[[saml]]
+== SAML
+
+External Authentication on the Appliance has been enhanced to support SAML.
+The SAML implementation has been tested with KeyCloak 1.8.
+
+In this guide we will cover how to manually configure a 7.2 Appliance's
```

Comment:
Ah ok, sorry, this just means RHEL 7.2, but that's just upstream.  I'll update. Thanks.